runeq/v2sdk/stream.py

```diff
@@ -0,0 +1,60 @@
+"""
+V2 SDK functionality to support Stream operations.
+
+"""
+from runeq import errors
+from . import client
+
+
+def get_stream(stream_id: str, **params):
+    """
+    Fetch stream data from the Stream API.
+
+    Parameters
+    ----------
+    stream_id : str
+        Unique string identifier for a stream.
+
+    **params
+        limit: Limits the number of points returned for the stream.
+        format: 'csv' or 'json'. Defaults to csv.
+
+    Returns
+    -------
+    Generator over stream data in the specified format, defaulting to csv.
```

Comment:
I think this could be documented a bit more clearly (with the same comment for the availability method): With CSV, each iteration will return a string for the response worth of CSV, whereas with JSON it comes back as a dictionary containing the the response body. In other words, the default is to returned unstructured data that would require further parsing to use, whereas specifying JSON returns structured data (though you still need to read the API docs to know what that structure is).

Reply:
Adding better documentation, but I am a little confused what you mean by the CSV response being unstructured data and the JSON response being structured. The csv data should be structured because it contains the column names in every response.

Reply:
Ahh I see what you mean now, made the change!